cosmwasm/submit-proposal.js

```diff
@@ -271,40 +240,67 @@ const programHandler = () => {
 
     program.name('submit-proposal').description('Submit governance proposals');
 
-    addAmplifierOptions(program);
-
-    program.addOption(new Option('-t, --title <title>', 'title of proposal').makeOptionMandatory(true));
-    program.addOption(new Option('-d, --description <description>', 'description of proposal').makeOptionMandatory(true));
-    program.addOption(new Option('--deposit <deposit>', 'the proposal deposit').makeOptionMandatory(true));
-    program.addOption(
-        new Option('-r, --runAs <runAs>', 'the address that will execute the message. Defaults to governance address').default(
-            governanceAddress,
-        ),
-    );
-    program.addOption(
-        new Option('--proposalType <proposalType>', 'proposal type')
-            .choices(['store', 'storeInstantiate', 'instantiate', 'execute', 'paramChange'])
-            .makeOptionMandatory(true),
-    );
-
-    program.addOption(new Option('--source <source>', "a valid HTTPS URI to the contract's source code"));
-    program.addOption(
-        new Option('--builder <builder>', 'a valid docker image name with tag, such as "cosmwasm/workspace-optimizer:0.16.0'),
-    );
-    program.addOption(
-        new Option('-i, --instantiateAddresses <instantiateAddresses>', 'comma separated list of addresses allowed to instantiate'),
-    );
-
-    program.addOption(new Option('--msg <msg>', 'json encoded message to submit with an execute contract proposal'));
-
-    program.addOption(new Option('--changes <changes>', 'parameter changes'));
+    const storeCmd = program
+        .command('store')
+        .description('Submit a wasm binary proposal')
+        .action((options) => {
+            mainProcessor(storeCode, options);
+        });
+    addAmplifierOptions(storeCmd, {
+        contractOptions: true,
+        storeOptions: true,
+        storeProposalOptions: true,
+        proposalOptions: true,
+        runAs: true,
+    });
 
-    program.addOption(new Option('--predictOnly', 'output the predicted changes only').env('PREDICT_ONLY'));
+    const storeInstantiateCmd = program
+        .command('store-instantiate')
```

Comment:
Could you please rename this command to camelCase to keep it consistent with other scripts? Here're [an example subcommands](https://github.com/axelarnetwork/axelar-contract-deployments/blob/fc080fa8d0e003cbf4427f8125d58557506b0589/sui/operators.js#L190-L214) in sui scripts.

Update instruction in the `README.md` file as well.